superset-frontend/src/views/RootContextProviders.tsx

```diff
@@ -35,35 +36,48 @@ const { common } = getBootstrapData();
 
 const extensionsRegistry = getExtensionsRegistry();
 
+const antdTheme = {
+  components: {
+    Badge: {
```

Comment:
Some GPT inputs that may help when defining the pattern for global vs component properties.

<img width="968" alt="Screenshot 2024-06-17 at 12 04 02" src="https://github.com/apache/superset/assets/70410625/ef1fb575-07e2-4c09-aabf-d83adaeaaef5">
<img width="941" alt="Screenshot 2024-06-17 at 12 04 40" src="https://github.com/apache/superset/assets/70410625/0bf4ea64-5529-48d7-9d33-7ec80a16ce31">